openlibrary/macros/LoanStatus.html

```diff
@@ -25,81 +30,69 @@
 
 $ borrow_link = page.url() + '/borrow'
 
-<div class="btn-notice" id="read-options">
-  $if user_loan:
-      <p>
-        <a href="$borrow_link" title="Borrow from $contributor"
-           data-userid="$(user_loan['userid'])" id="read_ebook"
-           class="borrow-btn borrow-link">Read eBook</a>
-      </p>
-      $ return_url = page.url().rsplit('/', 1)[0] + '/do_return/borrow'
-      <form action="$return_url" method="post" class="waitinglist-form return-book">
-        <input type="hidden" name="action" value="return" />
-        <input type="submit" value="Return eBook" class="submit unwait-btn" id="return_ebook"/>
-      </form>
-      $:macros.daisy(page)
+$if user_loan:
+    <p>
+      <a href="$borrow_link" title="Borrow from $contributor"
+         data-userid="$(user_loan['userid'])" id="read_ebook"
+         class="borrow-btn borrow-link">Read eBook</a>
+    </p>
+    $ return_url = page.url().rsplit('/', 1)[0] + '/do_return/borrow'
+    <form action="$return_url" method="post" class="waitinglist-form return-book">
+      <input type="hidden" name="action" value="return" />
+      <input type="submit" value="Return eBook" class="submit unwait-btn" id="return_ebook"/>
+    </form>
+
+$elif (availability == 'borrow_available') or my_turn_to_borrow:
+    <p>
+      <a href="$borrow_link" title="Borrow from $contributor" id="borrow_ebook" class="borrow-btn borrow-link">
```

Comment:
QUESTION: Do we need to html-escape and/or quote-escape `$contributor`?

Reply:
No -- if we ran `$:` this would render the block as html. Using `$` by itself should safely escape